spring-cloud-sleuth-core/src/main/java/org/springframework/cloud/sleuth/instrument/messaging/AbstractTraceChannelInterceptor.java

```diff
@@ -20,6 +20,9 @@
  */
 abstract class AbstractTraceChannelInterceptor extends ChannelInterceptorAdapter implements ExecutorChannelInterceptor {
 
+	/**
+	 * A default prefix for span name for discerning messaging origin of the span
```

Comment:
Fixed in https://github.com/spring-cloud/spring-cloud-sleuth/commit/7c7043910caad26e863fca7a820c6aefb2d6934d